draft-ietf-oauth-jwt-introspection-response.xml

```diff
@@ -190,179 +190,176 @@
+     ("type") header to "token-introspection+jwt".</t>
+
+     <t>The JWT MUST include the following top-level claims:
+     <list hangIndent="8" style="hanging">
+         <t hangText="iss">MUST be set to the issuer URL of the authorization
+         server.</t>
+         <t hangText="aud">MUST identify the resource server receiving the token
+         introspection response.</t>
+         <t hangText="iat">MUST be set to the time when the introspection response
+         was created by the authorization server.</t>
+         <t hangText="token_introspection">A JSON object containing the members
+         of the token introspection response, as specified in the "OAuth Token
+         Introspection Response" registry established by <xref target="RFC7662"/>
+         as well as other members. The separation of the introspection members into
+         a dedicated containing JWT claim is intended to prevent conflict and confusion
+         with top-level JWT claims that may bear the same name.
+         <vspace blankLines="1" />
+         If the access token is invalid, expired, revoked, or is not intended for the
+         calling resource server (audience), the authorization server MUST set the value of the
+         <spanx style="verb">active</spanx> member in the <spanx style="verb">token_introspection</spanx>
+         claim to <spanx style="verb">false</spanx> and other members MUST NOT be included.
+         Otherwise, the <spanx style="verb">active</spanx> member is set to <spanx style="verb">true</spanx>.
+         <vspace blankLines="1" />
+         If possible, the AS MUST narrow down the <spanx style="verb">scope</spanx> value
+         to the scopes relevant to the particular RS.
+         <vspace blankLines="1" />
+         Claims from the "JSON Web Token Claims" registry that are commonly used in
+         <xref target="OpenID.Core"/> and can be applied to the resource owner MAY be included
+         as members in the <spanx style="verb">token_introspection</spanx> claim.
+         They can serve to convey the privileges delegated to the client,
+         to identify the resource owner as a natural person or to
+         provide a required contact detail, such as an e-Mail address or phone number.
+         When transmitting such claims the AS acts as an identity provider in regard to
+         the RS. The AS determines based on its RS-specific policy what claims about the
+         resource owner to return in the token introspection response.
+         <vspace blankLines="1" />
+         The AS MUST ensure the release of any privacy-sensitive data is legally based.
+         <vspace blankLines="1" />
+         Further content of the introspection response is determined by the RS-specific
+         policy at the AS.</t>
+     </list>
+     </t>
+
+     <t>The JWT MAY include other claims, including those from the "JSON Web Token Claims"
+     registry established by <xref target="RFC7519"/>, such as <spanx style="verb">iat</spanx>
```

Comment:
current text (above) makes "iat" a mandatory claim, I suggest this text is removed